### ISSUE TITLE
Item Pickup Log on Armor

### ISSUE THREAD

**Issue by MCGamer00000** [BiscuitDevelopment/SkyblockAddons#165]
Whenever you put on armor, it says -1 armor name, and when you take it off, it says +1 armor name. 
Maybe include the armor as part of the inventory whenever checking for item differences.

Also, as a suggestion, maybe make the log include coins, as the system that hypixel has for telling whenever you get coins is very bad.

**Comment by DidiSkywalker** [comment]
Coins are displayed in the scoreboard next to your purse now right? I think that's pretty good and would only clutter the pickup log.
But armor slots should probably be included, yea.

**Comment by MCGamer00000** [comment]
Yes, it is included now, but it is much like the action bar system, sometimes it increases more than the s coreboard says it increases. Like Scavenger and telekinesis set the scoreboard at the same time, and only one shows.
Anyway, your right that it would clutter. Let's just forget about the suggestion. 
The point of this issue is to include the armor slots.

**Comment by ViralTaco** [comment]
> Also, as a (possible) suggestion, maybe make the log include coins, as the system that hypixel has for telling whenever you get coins is very bad.

That's actually due to how minecraft was made, lol. 
At least for 1.8.9 from Forge. Just look at net.minecraft.item.ItemStack
While it's possible to implement it doesn't seem like the most urgent of issues.

**Comment by ichbineinNerd** [comment]
I cant seem to manage to set up my development environment correctly so I cant test it, but that should be fixed by changing the line 

    main.getInventoryUtils().getInventoryDifference(p.inventory.mainInventory); //PlayerListener.java L349

to

    ItemStack[] combinedInventory = new ItemStack[p.inventory.mainInventory.length + p.inventory.armorInventory.length];
    System.arraycopy(p.inventory.mainInventory, 0, combinedInventory, 0, p.inventory.mainInventory.length);
    System.arraycopy(p.inventory.armorInventory, 0, combinedInventory, p.inventory.mainInventory.length, p.inventory.armorInventory.length);
    main.getInventoryUtils().getInventoryDifference(combinedInventory);
